apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -435,6 +447,16 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
 													  }).toString()}`
 													: ''
 											}`}
+											onClick={() => {
+												if (!coinBalance) {
+													return;
+												}
+
+												ampli.clickedSwapCoin({
```

Comment:
might be nice to add the `sourceFlow` property to this event so we can differentiate clicks from the large button vs clicks from individual token rows